css/responsive/tablet.styl

```diff
@@ -41,11 +41,8 @@ for col in 1..$cols
 // ARCHIVE EDITORIAL
 
 #posts-archive-editorial
-  height: 100%
-  min-height: 600px
   .col
-    height: 100%
-    min-height: 600px
+    min-height: 620px
```

Comment:
well the images can be as big as 620px high so if they are and they are not scaled [no scaling or funk happening in this branch] then the page will have to overflow window